src/main/java/edu/cornell/kfs/sys/document/service/impl/CuPaymentSourceHelperServiceImpl.java

```diff
@@ -0,0 +1,78 @@
+package edu.cornell.kfs.sys.document.service.impl;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang.WordUtils;
+import org.kuali.kfs.pdp.PdpParameterConstants;
+import org.kuali.kfs.pdp.businessobject.PaymentNoteText;
+import org.kuali.kfs.sys.document.service.impl.PaymentSourceHelperServiceImpl;
+import org.kuali.kfs.sys.service.impl.KfsParameterConstants;
+import org.kuali.rice.core.api.util.type.KualiInteger;
+
+import edu.cornell.kfs.fp.document.CuDisbursementVoucherConstants;
+
+public class CuPaymentSourceHelperServiceImpl extends PaymentSourceHelperServiceImpl {
+    
+    @Override
+    public List<PaymentNoteText> buildNotesForCheckStubText(String text, int previousLineCount) {
+        PaymentNoteText pnt = null;
```

Comment:
This code is a customized version of that from PaymentSourceHelperServiceImpl.buildNotesForCheckStubText, but I'll see if I can clean it up.

Reply:
It sounded  from the standup that is existing customization that got moved into a new class.  I am fine leaving as is.  Let me know if you want to pursue the cleanup effort.

Reply:
Based on our discussion at the standup, it sounds like it would be best to leave this as-is for now, since it's basically a cut-and-paste of an existing customization. If it's okay with you and with Daniela, we can move forward with this code as-is, and then perform cleanup and bug fixing in a future PR.